### PR TITLE
Review and simplify LSP1 Accept / Reject Assets guide

### PR DESCRIPTION
**Pull request by CJ42**

### CODE REVIEW

File: docs/learn/universal-receiver/accept-reject-assets.md [hunk 1]
```diff
@@ -8,35 +8,39 @@ import TabItem from '@theme/TabItem';
...
-The first step is to navigate to **[Remix's website](https://remix.ethereum.org/)** and create a new solidity file under the **contracts** folder.
+1. First go to the **[Remix's website](https://remix.ethereum.org/)**. Create a new solidity file `UniversalReceiverDelegate.sol` under the **contracts** folder.
 
 ![Creating Universal Receiver Delegate in Remix](/img/guides/lsp1/remix-creating-file.jpeg)
 
-After creating the **UniversalReceiverDelegate.sol** file, copy the code snippet below inside the file created. This code snippet will be responsible for rejecting all LSP7 & LSP8 assets being transferred to your profile.
+2. Copy the code snippet below inside the file. It contains the logic for rejecting any LSP7 & LSP8 assets being transferred to the Universal Profile.
 
-```sol title="UniversalReceiverDelegate.sol - Solidity Code snippet of the URD that reject all assets"
+```solidity title="UniversalReceiverDelegate.sol - Solidity Code snippet of the URD that reject any assets"
```
Comment (YamenMerhi) [inline docs/learn/universal-receiver/accept-reject-assets.md:43]: I ran the code, it's not compiling, maybe it's related to the change of the package?

**Review by YamenMerhi — APPROVED** [review]
LGTM (Just needs to check the sol code)

File: docs/learn/universal-receiver/accept-reject-assets.md (hunk 1 above)
Comment (CJ42) [reply docs/learn/universal-receiver/accept-reject-assets.md:43]: Strange I tried on Remix on my end. It compiled successfully

**Comment by CJ42** [comment]
@YamenMerhi will merge and re-open a PR if the issue arise. Compiler error seems to be a false positive.

**Pull request merged**